src/fluree/db/nameservice/storage.cljc

```diff
@@ -19,62 +20,93 @@
       (storage/build-address ledger-alias)))
 
 (defn ns-record
-  "Generates nameservice metadata map for JSON storage. For now, since we only
-  have a single branch possible, always sets default-branch. Eventually will
-  need to merge changes from different branches into existing metadata map"
-  [ns-address {address "address", alias "alias", branch "branch", :as commit-jsonld}]
-  (let [branch-iri (str ns-address "(" branch ")")]
-    {"@context"      iri/fluree-context-url
-     "@id"           ns-address
-     "defaultBranch" branch-iri
-     "ledgerAlias"   alias
-     "branches"      [{"@id"     branch-iri
-                       "address" address
-                       "commit"  commit-jsonld}]}))
+  "Generates nameservice metadata map for JSON storage using new minimal format"
+  [ledger-alias branch commit-address t index-address]
+  (let [branch (or branch "main")]
+    (cond-> {"@context"     {"f" iri/f-ns}
+             "@id"          (str ledger-alias "@" branch)
+             "@type"        ["f:Database" "f:PhysicalDatabase"]
+             "f:ledger"     {"@id" ledger-alias}
+             "f:branch"     branch
+             "f:commit"     {"@id" commit-address}
+             "f:t"          t
+             "f:status"     "ready"}
+      index-address (assoc "f:index" {"@id" index-address}))))
 
 (defn get-commit
+  "Returns the minimal nameservice record."
   ([record]
    (get-commit record nil))
-  ([record branch]
-   (log/info "Fetching commit from record:" record)
-   (let [branch-iri (if branch
-                      (str (get record "@id") "(" branch ")")
-                      (get record "defaultBranch"))]
-     (some #(when (= (get % "@id") branch-iri)
-              (get % "commit"))
-           (get record "branches")))))
+  ([record _branch]
+   ;; Always return the record itself for new format
+   record))
```

Comment:
Shouldn't we just remove this `get-commit` function all together? It isn't used anywhere outside this namespace, and the usage in here could easily also be removed to just use the record directly.